src/shim/shim-config.ts

```diff
@@ -325,6 +329,18 @@ class Config implements IConfig {
             }
         }
 
+        if (this.enableSendLiveMetrics === false) {
```

Comment:
This kind of check would be broken the moment we change default values, we need to improve the logic here

Reply:
Updated logic to first check if the value is a boolean.